aioftp/client.py

```diff
@@ -725,15 +735,19 @@ async def _new_stream(cls, local_path):
                 if raw_command in [None, "MLSD"]:
                     try:
                         command = ("MLSD " + str(cls.path)).strip()
-                        return await self.get_stream(command, "1xx")
+                        return await self.get_stream(
+                            command, "1xx", passive_commands=passive_commands
+                        )
```

Comment:
For this project it is better to use this style for making "too long calls":
``` python
return await self.get_stream(
    command, 
    "1xx", 
    passive_commands=passive_commands,
)
```

Reply:
:+1: Fixed it.